niaiddata.org/portal/gitops.json

```diff
@@ -192,65 +189,154 @@
       },
       "dataset": {
         "chartType": "pie",
-        "title": "Dataset"
+        "title": "Resources",
+        "chartRow": 0
+      },
+      "data_format": {
+        "chartType": "bar",
+        "title": "Data Format",
+        "chartRow": 0
+      },
+      "data_type": {
+        "chartType": "pie",
+        "title": "Data Type",
+        "chartRow": 0
+      },
+      "experimental_strategies": {
+        "chartType": "bar",
+        "title": "Experimental Strategies",
+        "chartRow": 0
       },
       "species": {
         "chartType": "bar",
-        "title": "Genus species"
+        "title": "Genus species",
+        "chartRow": 0
       },
       "gender": {
         "chartType": "pie",
-        "title": "Gender"
-      }
-    },
-    "projectSections": [
-      { 
-        "title": "Resource",
-        "field": "dataset",
-        "options": []
-      }
-    ], 
-    "subjectSections": [
-      { 
-        "title": "Ethnicity", 
-        "field": "ethnicity",
-        "options": []
-      },
-      { 
         "title": "Gender",
-        "field": "gender",
-        "options": []
+        "chartRow": 1
       },
-      { 
-        "title": "Genus species",
-        "field": "species",
-        "options": []
+      "race": {
+        "chartType": "pie",
+        "title": "Race",
+        "chartRow": 1
+      },
+      "ethnicity": {
+        "chartType": "pie",
+        "title": "Ethnicity",
+        "chartRow": 1
+      },
+      "biospecimen_anatomic_site": {
+        "chartType": "pie",
+        "title": "Biospecimen Anatomic Site",
+        "chartRow": 1
       }
-    ],
+    },
     "fieldMapping" : [
       { "field": "dataset", "name": "Resource" },
       { "field": "studyAccession", "name": "Study" },
       { "field": "phenotype", "name": "Phenotype" },
       { "field": "gender", "name": "Gender" },
       { "field": "ethnicity", "name": "Ethnicity" },
       { "field": "strain", "name": "Strain" },
-      { "field": "species", "name": "Genus species" },
-      { "field": "submitter_id", "name": "Submitter ID" }
+      { "field": "species", "name": "Genus Species" },
```

Comment:
Species should be lower case here. It is a scientific naming convention, ie. _Homo sapiens_